Banking/src/main/java/com/example/Banking/dao/UserDao.java

```diff
@@ -0,0 +1,17 @@
+package com.example.Banking.dao;
+
+import com.example.Banking.exception.IdNotFoundException;
+import com.example.Banking.model.Account;
+import com.example.Banking.model.UserTransaction;
+
+import java.util.Optional;
+
+
+public interface UserDao {
```

Comment:
I don't understand the purpose of this Dao. Its named as UserDao but performing CRUD on Account. While we have a Dao https://github.com/khoros-campus-hires/BankingSystem/pull/2/files?file-filters%5B%5D=.java&file-filters%5B%5D=.properties#diff-703d1ff85ba427421b411aeac6f64bf28396b97a9bedd7dddbc29c39021d73ac for Account

Reply:
The class names were created before and is not in proper sync with the program. I got it, Sorry for those. will change and review one time before pushing.

Reply:
We need a `TransactionDao` which can create and fetch the transaction

Reply:
No Account or User DB operation should be performed within `TransactionDao`